src/main/scala/hevs/graphics/FunGraphics.scala

```diff
@@ -282,33 +283,103 @@ class FunGraphics(val width: Int, val height: Int, val xoffset: Int, val yoffset
     g2d.fillOval(posX, posY, width, height)
   }
 
-  override def drawString(posX: Int, posY: Int, str: String): Unit = {
-    g2d.drawString(str, posX, posY)
+  override def getStringSize(str: String, font: Font): Rectangle2D = {
```

Comment:
Should document that it returns the bouding box of the text

Reply:
Like the docstring in Graphics.scala:184 ?